internal/stack/_static/docker-compose-stack.yml.tmpl

```diff
@@ -112,6 +112,10 @@ services:
       - "../certs/fleet-server:/etc/ssl/elastic-agent"
     ports:
       - "127.0.0.1:8220:8220"
+      {{ $apm_server_enabled := fact "apm_server_enabled" }}
+      {{ if eq $apm_server_enabled "true" }}
```

Comment:
This should be exposed from `elastic-agent` container not `fleet-server` right? APM should run in the same container as Agent, or I miss sth here?

Reply:
this is an elastic-agent container too.  this ended up with the following containers:

```
a6d36785b0be   docker.elastic.co/elastic-agent/elastic-agent-complete:8.10.1   "/usr/bin/tini -- /u…"   20 minutes ago   Up 19 minutes (healthy)                                                          elastic-package-stack-apm-elastic-agent-1
c237ccac4ba7   docker.elastic.co/elastic-agent/elastic-agent-complete:8.10.1   "/usr/bin/tini -- /u…"   20 minutes ago   Up 20 minutes (healthy)     127.0.0.1:8200->8200/tcp, 127.0.0.1:8220->8220/tcp   elastic-package-stack-apm-fleet-server-1

```

Reply:
Ah ok I see, it makes sense since it will start either way.